### DISCUSSION TITLE
Help text for hierarchical option on taxonomies

### DISCUSSION THREAD

**Discussion by pglewis** [pods-framework/pods#6829] [Q&A]
We cover this in a few sections in the online docs... but hey, real coders don't read docs, right?

**Comment by jimtrue** [comment]
@pglewis @sc0ttkclark @JoryHogeveen While I love these items, how do you easily find those sections in the code where the help text is to be added? I'd love to handle these and jump on them quickly, but might need a little guidance to quickly find them and also to have quickly documented the correct format for adding help text to a field in our admin.

**Comment by JoryHogeveen** [comment]
I don't get this issue. The hierarchical option has a help text. Do we need to change that?
